recipe/meta.yaml

```diff
@@ -31,11 +31,13 @@ requirements:
 
 test:
   requires:
-    - nose
+    - pytest
+    - scipy
   imports:
     - slycot
   commands:
-    - python -c "from slycot import test; test()"
```

Comment:
So, does this command not work anymore?

Reply:
Seems so. I did not notice that before. The command imports Tester from numpy which uses nose...

Off to file a bug in the Slycot repository and working on a PR.

Reply:
Done in https://github.com/python-control/Slycot/pull/136

Found https://github.com/numpy/numpy/pull/10827, which means numpy did the same transition and newer numpy.Tester would call pytest again.